examples/demo/gtk-layer-demo.c

```diff
@@ -170,6 +172,39 @@ anchor_option_callback (const gchar *_option_name, const gchar *value, void *_da
     return TRUE;
 }
 
+gboolean
+keyboard_option_callback (const gchar *_option_name, const gchar *value, void *_data, GError **error)
+{
+    (void)_option_name;
+    (void)_data;
+
+    if (!value) {
+        /* without argument = exclusive (retain old behavior) */
+        default_keyboard_interactivity = GTK_LAYER_SHELL_KEYBOARD_EXCLUSIVE;
+        return TRUE;
+    }
+    if (value[0] == '0' && value[1] == 0) {
+        default_keyboard_interactivity = GTK_LAYER_SHELL_KEYBOARD_NONE;
+    } else if (value[0] == '1' && value[1] == 0) {
+        default_keyboard_interactivity = GTK_LAYER_SHELL_KEYBOARD_EXCLUSIVE;
+    } else if (value[0] == '2' && value[1] == 0) {
```

Comment:
0/1 could be handled to retain old behavior, but I wonder if it makes sense to handle 2. Using enum names is clearer.

Reply:
The numbers seem unnecisarry. Just the names should be fine.